pkgs/applications/version-management/git-stack/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
```

```suggestion
{ lib
, stdenv
```

Because it's (an unwritten) standard.